packages/iframe-coordinator-cli/src/views/IframeEmbed.vue

```diff
@@ -5,6 +5,8 @@
       <span class="app-route">{{ frameRoute }}</span>
       as
       <span class="frame-url">{{ frameUrl }}</span>
+      with Client Id of
+      <span class="client-id">{{ currentClientId }}</span>
```

Comment:
showing current client id so it's easier to see and test whether the correct client is loaded